src/framework/xr/xr-view.js

```diff
@@ -324,6 +324,14 @@ class XrView extends EventHandler {
         return this._viewInvOffMat;
     }
 
+    /**
+     * @type {Mat4}
+     * @ignore
+     */
+    get viewInvMat() {
```

Comment:
I know you've set this as `@ignore` but I don't like the name if this is intended as API. `invViewMat` is better. I've head the phrase 'inverse view matrix' before. I've never heard anyone say 'view inverse matrix'.

Reply:
Also, the naming isn't consistent with the `CameraComponent`'s `viewMatrix` and `projectionMatrix` properties.

Reply:
Okay sounds like this is a bigger refactor ill close this PR and create at a later date